tests/tests/swfs/avm1/asnative/output.txt

```diff
@@ -0,0 +1,30 @@
+// ASnative(100)
+undefined
+
+// ASnative(100, 0)
+[type Function]
+
+// ASnative(100, "string")
+[type Function]
+
+// ASnative(100, 0, 0)
+undefined
+
+// ASnative("100", "0")
+[type Function]
+
+// ASnative("onehundred", 0)
+undefined
+
+// ASnative(100, {valueOf: function() {return 4;}})("success")
+success
+
+// ASnative(100, 4.5)("success")
+success
```

Comment:
Could you also add tests for `ASnative(100, 0x1_0000 + 4)` and `ASnative(100, 0x1_0000_0000 + 4)`, to check overflow behavior?